content/kubermatic/master/architecture/role_based_access_control/_index.en.md

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Role based access control"
```

Comment:
I know you did not change this, but can we update this to be title case as well while we're at it?

```suggestion
title = "Role-based Access Control"
```